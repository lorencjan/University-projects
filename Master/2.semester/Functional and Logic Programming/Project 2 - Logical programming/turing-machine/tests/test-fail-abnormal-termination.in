S a S R
S b S R
aaaaaabbaaaabbbbbbbbbb
