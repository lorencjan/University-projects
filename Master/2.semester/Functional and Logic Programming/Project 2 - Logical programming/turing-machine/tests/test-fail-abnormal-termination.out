Runtime error: Abnormal TM termination.